common/persistence/persistence-tests/persistenceTestBase.go

```diff
@@ -453,8 +448,5 @@ func GenerateRandomDBName(n int) string {
 
 func timeComparator(t1, t2 time.Time, timeTolerance time.Duration) bool {
 	diff := t2.Sub(t1)
-	if diff.Nanoseconds() <= timeTolerance.Nanoseconds() {
-		return true
-	}
-	return false
+	return diff.Nanoseconds() <= timeTolerance.Nanoseconds()
```

Comment:
yeah I think it can. I didn't pay attention to what the code is actually doing when fixing the lint warning. 
I can remove the nanoseconds call in other prs.